### PR TITLE
Make LayoutKit extension safe (#43)

### PR DESCRIPTION
**Pull request by nicksnyder**

### CODE REVIEW

File: LayoutKit/UIKitSupport.swift [hunk 1]
```diff
@@ -25,4 +23,22 @@ extension UIView {
...
-}
\ No newline at end of file
+
+    var userInterfaceLayoutDirection: UIUserInterfaceLayoutDirection {
+        if #available(iOS 9.0, *) {
+            return UIView.userInterfaceLayoutDirectionForSemanticContentAttribute(semanticContentAttribute)
+        } else {
+            #if LAYOUTKIT_EXTENSION_SAFE
+                // Not aware of any API that we can use on iOS 8.0 in an extension
+                // that would allow us to know
```
Comment (nicksnyder) [inline LayoutKit/UIKitSupport.swift]: fix this comment

Comment (FabianTerhorst) [inline LayoutKit/UIKitSupport.swift]: you can also get the direction from the current language: 
```
if NSLocale.characterDirectionForLanguage(NSLocale.currentLocale().objectForKey(NSLocaleLanguageCode) as! String) == .LeftToRight {

}
```

Comment (nicksnyder) [inline LayoutKit/UIKitSupport.swift]: Thanks for the suggestion, that is much better.

**Comment by plivesey** [comment]
LGTM

**Pull request merged**